docs/eks-jobs-migration.md

```diff
@@ -0,0 +1,216 @@
+  - You can check out the following ["Prow Results"][prow-results-default] link
+    for recent jobs that are running in the `default` cluster
+  - For explanation reasons, let's say that you picked up a job called
+    `pull-jobset-test-integration-main`
+- Edit the file that `pull-jobset-test-integration-main` is defined in. All
+  jobs are defined in the [kubernetes/test-infra][k-test-infra] repository. You
+  can use GitHub or our [Code Search tool][cs-test-infra] to find the file
+  where this job is defined (search by job name).
+- Look for a `cluster:` key in the job definition. If there isn't one or it's
+  set to `default`, then the job runs in the default cluster. Add (or replace)
+  the following `cluster:` key: `cluster: eks-prow-build-cluster`.
+  - **IMPORTANT: if you see any entries under `label` that says `gce` skip this
+    job and get back to it the next time as this is not ready to be moved yet.**
+  - **IMPORTANT: if you see any entries under `label` that might indicate that
+    a job is using some external resource (e.g. some cloud platform), you need
+    to check if the Community Ownership criteria is satisfied for that resource
+    (see the Community Ownership section for more details)**
+  - **IMPORTANT: jobs running in community-owned clusters must have resource
+    requests and limits specified for cluster-autoscaler to work properly.
+    If that's not the case for your job, please see the next section for some
+    details about determining correct resource requests and limits.**
+  - Here's an [example of a job][example-eks-job] as a reference (pay attention
+    to the `cluster:` key)
+- Save the file, commit the change, create a branch and file a PR
+- Once the PR is merged, follow the guidelines in the Post Migration Period
+  section of this document to ensure that the job remains stable
+
+If you have any trouble, please leave a note in the following
+[GitHub issue][test-infra-gh-issue] or come to
+[#sig-k8s-infra][slack-k8s-infra] or [#sig-testing][slack-sig-testing] Slack
+channel to ask for help.
+
+[prow-results-default]: https://prow.k8s.io/?cluster=default
+[cs-test-infra]: https://cs.k8s.io/?q=job-name-here&i=nope&files=config%2F&excludeFiles=&repos=kubernetes/test-infra
+[example-eks-job]: https://github.com/kubernetes/test-infra/blob/1d219efcca8a254aaca2c34570db0a56a05f5770/config/jobs/kubernetes/cloud-provider-aws/cloud-provider-aws-config.yaml#L3C1-L31
+[test-infra-gh-issue]: https://github.com/kubernetes/test-infra/issues/29722
+[slack-k8s-infra]: https://kubernetes.slack.com/archives/CCK68P2Q2
+[slack-sig-testing]: https://kubernetes.slack.com/archives/C09QZ4DQB
+
+### Determining Resource Requests and Limits
+
+Jobs running in community-owned clusters must have resource requests and 
+limits specified for cluster-autoscaler to work properly. However, determining
+this is not always easy and incorrect requests and limits can cause the job
+to start flaking or failing.
```

Comment:
Or even affect other jobs running in the cluster in case it causes CPU throttling.

Reply:
AFAIK, throttling only applies to a specific cgroupns and each pod is supposed to have its own namespace. That being said, hopefully, other jobs shouldn't be affected by throttling.